src/Psy/Configuration.php

```diff
@@ -772,6 +774,26 @@ public function getLoop()
     }
 
     /**
+     * Enables tab completion mode
+     *
+     * @param $tabCompletion
+     */
+    public function setTabCompletion($tabCompletion)
+    {
+        $this->tabCompletion = (bool) $tabCompletion;
```

Comment:
We should check whether the Readline implementation supports tab completion rather than setting one up and having it just sit there :)